lib/deflate.js

```diff
@@ -143,6 +144,7 @@ function Deflate(options) {
   this.msg    = '';     // error message
   this.ended  = false;  // used to avoid multiple onEnd() calls
   this.chunks = [];     // chunks of compressed data
+  this._dict_set = false; // have we set the dictionary
```

Comment:
Search for this var name in editor. It's used in constructor only. `deflateSetHeader()` is also colled in constructor only.

I don't know what you meaned, but this code is not used in push anyhow. At least, now.